filer/admin/folderadmin.py

```diff
@@ -25,26 +25,40 @@
 from django.utils.translation import ugettext as _
 from django.utils.translation import ungettext, ugettext_lazy
 
-from filer import settings
-from filer.admin.forms import (CopyFilesAndFoldersForm, ResizeImagesForm,
-                               RenameFilesForm)
-from filer.admin.permissions import PrimitivePermissionAwareModelAdmin
-from filer.admin.patched.admin_utils import get_deleted_objects
-from filer.admin.tools import (userperms_for_request,
-                               check_folder_edit_permissions,
-                               check_files_edit_permissions,
-                               check_files_read_permissions,
-                               check_folder_read_permissions,
-                               admin_each_context)
-from filer.models import (Folder, FolderRoot, UnfiledImages, File, tools,
-                          ImagesWithMissingData, FolderPermission, Image)
-from filer.settings import FILER_PAGINATE_BY
-from filer.thumbnail_processors import normalize_subject_location
-from filer.utils.compatibility import (
-    get_delete_permission, quote, unquote, capfirst)
-from filer.utils.filer_easy_thumbnails import FilerActionThumbnailer
-from filer.views import (popup_status, popup_param, selectfolder_status,
-                         selectfolder_param)
+from .. import settings
+from .forms import (
+    CopyFilesAndFoldersForm,
+    ResizeImagesForm,
+    RenameFilesForm,
+)
+from .permissions import PrimitivePermissionAwareModelAdmin
+from .patched.admin_utils import get_deleted_objects
+from .tools import (
+    userperms_for_request,
+    check_folder_edit_permissions,
+    check_files_edit_permissions,
+    check_files_read_permissions,
+    check_folder_read_permissions,
+    admin_each_context,
+)
+from .tools import (popup_status, admin_url_params_encoded,
+                    AdminContext)
+from . import views
+from ..models import (
+    FolderRoot,
+    ImagesWithMissingData,
+    UnfiledImages,
+    Folder,
+    File,
+    Image,
+    FolderPermission,
+    tools,
+)
+from ..settings import FILER_PAGINATE_BY
+from ..thumbnail_processors import normalize_subject_location
+from ..utils.compatibility import (
+    get_delete_permission, quote, unquote, capfirst, LTE_DJANGO_1_6)
+from ..utils.filer_easy_thumbnails import FilerActionThumbnailer
 
```

Comment:
Also here: imports should be "most remote" -> "most local" and alphabetized within their groupings. In Filer in particular, we had a lot of duplicate imports that had to be cleaned out, this happens when they're not alphabetised/organised.